src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrderReversalTransactionDto.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    internal class PaymentOrderReversalTransactionDto
+    {
+        public PaymentOrderReversalTransactionDto(PaymentOrderReversalTransaction transaction)
+        {
+            Amount = transaction.Amount.InLowestMonetaryUnit;
+            Description = transaction.Description;
+            if (transaction.OrderItems != null && transaction.OrderItems.Any())
+            {
+                OrderItems = new List<Sdk.OrderItemDto>();
+                foreach (var item in transaction.OrderItems)
+                {
+                    OrderItems.Add(new Sdk.OrderItemDto(item));
+                }
+            }
+            PayeeReference = transaction.PayeeReference;
+            VatAmount = transaction.VatAmount.InLowestMonetaryUnit;
+        }
+
+        public long Amount { get; }
```

Comment:
I see `Amount` is exposed as `long` here and `int` in other places. It should probably be `long` everywhere?

Reply:
That is correct. It was defined as `int` previously but `long` is the correct type.